tests/jms/test_jms.py

```diff
@@ -0,0 +1,90 @@
+# pylint: disable-all
+
+import json
+import os
+import threading
+import time
+import uuid
+from typing import Optional
+
+import pytest
+import stomp
+from pytest_mock import MockerFixture
+
+from jms.connector import subscribe_to_databus
+from recommender.types import UserAction
+from tests.factories.requests.user_actions import UserActionFactory
+
+JMS_HOST = os.environ.get("TEST_RS_SUBSCRIBER_HOST", "127.0.0.1")
+JMS_PORT = int(os.environ.get("TEST_RS_SUBSCRIBER_PORT", "61613"))
+JMS_USERNAME = os.environ.get("TEST_RS_SUBSCRIBER_USERNAME", "guest")
+JMS_PASSWORD = os.environ.get("TEST_RS_SUBSCRIBER_PASSWORD", "guest")
+JMS_TOPIC = os.environ.get("TEST_RS_SUBSCRIBER_TOPIC", "/topic/user_actions_test")
+
+Second = int
```

Comment:
Why it's done in such a way?

Reply:
This way you know that the argument is in seconds - that's  a pretty handy thing to use instead of documenting it in docstring or using names such as `time_ms` or `time_sec` you can write it as `time: Second` or `time: Milisecond`.
This is a convention though